src/Dispatcher/MASA.Contrib.Dispatcher.IntegrationEvents.Dapr/IntegrationEventHostedService.cs

```diff
@@ -2,19 +2,21 @@
 
 public class IntegrationEventHostedService : BackgroundService
 {
-    private readonly ILogger<IntegrationEventHostedService> _logger;
+    private readonly ILogger<IntegrationEventHostedService>? _logger;
     private readonly IProcessingServer _processingServer;
 
-    public IntegrationEventHostedService(ILogger<IntegrationEventHostedService> logger, IProcessingServer processingServer)
+    public IntegrationEventHostedService(IProcessingServer processingServer, ILogger<IntegrationEventHostedService>? logger)
     {
         _logger = logger;
         _processingServer = processingServer;
     }
 
     protected override Task ExecuteAsync(CancellationToken stoppingToken)
     {
-        _logger.LogDebug("----- IntegrationEvent background task is starting");
-        
+        Task.Delay(2000, stoppingToken);
```

Comment:
Why delay?

Reply:
Deleted, originally wanted to delay the execution of background tasks